lib.js

```diff
@@ -93,3 +93,29 @@ export const generateTestsuiteResult = (suiteResult) => {
 
   return result.join(EOL);
 };
+
+export const generateTestsuiteLogs = (suiteResult) => {
+  let result = [];
+  const testscases = suiteResult.testcase;
+
+  if (testscases) {
+    let errLogs = [...testscases.map(testcase => formatLogLine(testcase['system-err']))].filter(a => a !== '');
+    if (errLogs.length) {
+      result.push(EOL, '   > Error Log output:', EOL, EOL, '\t', ...errLogs);
+    }
+
+    let outLogs = [...testscases.map(testcase => formatLogLine(testcase['system-out']))].filter(a => a !== '');
+    if (outLogs.length) {
+      result.push(EOL, '   > Standard Log output:', EOL, EOL, '\t', ...outLogs);
+    }
+  }
+
+  return result.join('');
+};
+
+const formatLogLine = (logLine) => {
+  if (logLine) {
+    return logLine.join(EOL).split(EOL).join(EOL + '\t');
+  }
+  return '';
+};
```

Comment:
Nice, thanks. This is a good helper function. I can reuse it at the top in the other code that's already there. Meanwhile, I think this can be written as to make it more generically usable:
```suggestion
const formatLogLine = (log) => {
  if (log.join) {
    return log.join(EOL).split(EOL).join(EOL + '\t');
  }
  return log;
};
```
In this way, it can be used for both string and array logs.

Reply:
Updated 👍